libsrc/common/isascii.s

```diff
@@ -0,0 +1,27 @@
+; isascii.s
+;
+; This file is part of
+; cc65 - a freeware C compiler for 6502 based systems
+;
+; https://github.com/cc65/cc65
+;
+; See "LICENSE" file for legal information.
+;
+; int isascii (int c);
+;
+
+        .export         _isascii
+
+_isascii:
+        cpx     #$00            ; Char range ok?
+        bne     @L1             ; Jump if no
+
+        tay
```

Comment:
This is smaller and should be equivalent:
`  ASL
   TXA
   BNE @L1
   ADC #$FF
   RTS
  `

Reply:
Saved another byte:

`
_isascii:
        asl		a				; high-bit to carry
        txa 					        ; check range of input param
        beq		@L1			; out-of bounds?
        ldx		#$00			; yes, prepare returning of 'false'
        sec						; (will result in '0' due to next adc)
@L1:    adc 	#$FF			; calculate return value based on carry
        rts
`